library/src/main/java/com/bumptech/glide/util/LruCache.java

```diff
@@ -166,12 +168,14 @@ public void clearMemory() {
    */
   protected synchronized void trimToSize(int size) {
     Map.Entry<T, Y> last;
+    Iterator<Map.Entry<T, Y>> cacheIterator;
```

Comment:
just a preference (declare close to usage, and in scope): these two vars are only used within the loop, so they could be defined inside:
```
final Iterator<Map.Entry<T, Y>> cacheIterator  = cache.entrySet().iterator();
final Map.Entry<T, Y> last = cacheIterator.next();
```

Reply:
I agree in principle, but this matches the existing style, so let's just leave it as is for now.